File: javascript/models/review.js
function createReview(){
  let id = 0;
  return class Review{
    constructor(text, rest, cust){
      this.id = id++
      this.text = text
      this.restaurant = rest
      this.customer = cust
      Store.data.reviews.push(this)
    }
  }
}

File: object_relations/app/models/review.rb
class Review

  attr_accessor :review, :customer, :restaurant

  @@all = []

  def initialize(review, restaurant, customer)
    @review = review
    @restaurant = Restaurant.find_by_name(restaurant.name)
    @customer = customer
    @restaurant.reviews << review
    @@all << self
  end


end
